help/grandorgue.xml

```diff
@@ -7228,6 +7314,25 @@ attributes of a label.
      </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term>SetterCurrFileName</term>
+          <listitem>
+            <para>
+	      Represents the current combinations file name of the setter.
+	      The section must contain the GUI attributes of a label.
+	    </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Refresh, SetterPrevFile, SetterNextFile, SetterLoadCmb, SetterSaveCmb</term>
```

Comment:
There was already a Save button on the setter panel since before (that saved to a preset/settings file, right?). Why have you silently removed it from visibility by moving it to the general controls? Since the setter element Save is already available and could potentially have been used in sample sets, I understand that it shouldn't be removed lightly - but now you have a button with the same old name on the setter panel that does something slightly different. That could potentially be somewhat confusing. I think that changing to "Load cmb" and "Save cmb" as texts could be clearer - just as you renamed the old Save to "Save settings".

Reply:
Even I agree on that, shouldn't we use full term, so "Save combinations"?

Reply:
@rousseldenis Yes, it would be even better, but I think that the space on the drawstop might put a limit on what can be written, but maybe if the words are put on two lines with appropriate textbreakwidth set...

Reply:
> That could potentially be somewhat confusing. I think that changing to "Load cmb" and "Save cmb" as texts could be clearer - just as you renamed the old Save to "Save settings".

But `cmb` may be confused with the file name extension `.cmb` that is used for organ settings. ``Load yaml`` and ``Save yaml`` would not be understandable for most of non-technical people. Might be ``Load File`` and ``Save File``, because ``Prev File`` and ``Next File`` are present already?

Reply:
> Even if I agree on that, shouldn't we use full term, so "Save combinations"?

That long label wouldn't fit the button size.